### DISCUSSION TITLE
erorr when i use DatePickerModal

### DISCUSSION THREAD

**Discussion by gustavohbeg** [web-ridge/react-native-paper-dates#238] [General] [answered]
when i use DatePickerModal component i get the error: how resolve?
![Screenshot_20230110_173050_Expo Go](https://user-images.githubusercontent.com/82586205/211656234-4bee8665-3275-494d-85fe-5df684007929.jpg)

**Comment by iM-GeeKy** [comment, marked as answer]
You're error is a little vague, but as the error indicates you're missing `Intl`. Check this out in our documentation https://web-ridge.github.io/react-native-paper-dates/docs/intro#android-caveats.

@RichardLindhout Has also answered the same question [here](https://github.com/web-ridge/react-native-paper-dates/issues/213).
